package.json

```diff
@@ -29,12 +29,8 @@
         "integration-tests": "node -r source-map-support/register -r dotenv/config ./built/integration-tests/index.js dotenv_config_path=.env.integration-tests",
         "integration-tests:debug": "node -r source-map-support/register -r dotenv/config ./built/integration-tests/index.js dotenv_config_path=.env.integration-tests.debug",
         "test": "jest",
-        "db:setup": "dotenv npm run db:setup:run",
-        "db:setup:run": "psql \"$DATABASE_URL?sslmode=prefer\" -c 'CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\";'",
-        "db:setup:heroku": "psql \"$DATABASE_URL?sslmode=prefer\" -c 'CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\" WITH SCHEMA heroku_ext;'",
-        "db:reset": "prisma db push --skip-generate",
-        "db:reset-hard": "prisma db push --skip-generate --force-reset",
-        "db:reset-for-migration": "prisma migrate reset",
+        "db:reset": "prisma migrate reset --skip-generate",
+        "db:reset-hard": "prisma migrate reset -f --skip-generate",
```

Comment:
@Jonasdoubleyou Was there a special reason to use db push instead of migrate reset? 
When using the reset function we don't need the psql commands anymore. Review environments / integration tests and local setups seem to work.

Reply:
I think push takes the schema, so can be used to quickly check local changes to the schema. But I'm fine to change it and always require a migration to be generated first.

Reply:
We can add it back when we see that the new workflow is much more effort.